test/assertions.ts

```diff
@@ -1,13 +1,13 @@
 import { ElementDefinition } from 'cytoscape'
-import { existsSync } from 'fs'
+import { existsSync, lstatSync, readdirSync } from 'fs'
```

Comment:
Acá dice que esto no se usa
```suggestion
import { existsSync } from 'fs'
```